subsys/bluetooth/host/iso.c

```diff
@@ -818,6 +818,28 @@ void hci_le_cis_estabilished(struct net_buf *buf)
 	}
 
 	if (!evt->status) {
+		if (iso->role == BT_HCI_ROLE_PERIPHERAL) {
+			struct bt_iso_chan *chan = iso->iso.chan;
+			struct bt_iso_chan_io_qos *rx;
+			struct bt_iso_chan_io_qos *tx;
+
+			__ASSERT(chan != NULL && chan->qos != NULL,
+				 "Invalid ISO chan");
+
+			rx = chan->qos->rx;
+			tx = chan->qos->tx;
+
+			if (rx != NULL) {
+				rx->phy = evt->c_phy;
+				rx->sdu = evt->c_max_pdu;
+			}
+
+			if (tx != NULL) {
+				tx->phy = evt->p_phy;
+				tx->sdu = evt->p_max_pdu;
+			}
```

Comment:
Does phy and sdu already has a default value assigned, in case rx and/or tx are NULL?

Reply:
If rx/tx are NULL, then they cannot be assigned values. 

In case that they are non-NULL, they do not have a default value (the value is allocated by the application/upper layer). 

For central, these values shall be set when creating a CIG. 
For peripheral, these values may be set by the application, but will be overwritten by this.